src/Jobs/ResendMailJob.php

```diff
@@ -24,27 +24,60 @@ public function __construct(
         private array $to,
         private array $cc = [],
         private array $bcc = [],
+        private array $replyTo = []
     ) {
         //
     }
 
     public function handle(): void
     {
-        Mail::send([], callback: function (Message $message) {
-            $message->html($this->mail->html ?? '')
-                ->text($this->mail->text ?? '');
-
-            foreach ($this->mail->attachments as $attachment) {
-                $message->attachData($attachment->fileData, $attachment->filename, ['mime' => $attachment->mime]);
-            }
-
-            return $message
-                ->subject($this->mail->subject ?? '')
-                ->from($this->mail->from)
-                ->replyTo($this->mail->reply_to ?? [])
-                ->to($this->to ?? [])
-                ->cc($this->cc ?? [])
-                ->bcc($this->bcc ?? []);
+        Mail::send([], [], function (Message $message) {
+            $this->setMessageContent($message)
+                ->setMessageRecipients($message);
         });
     }
+
+    private function setMessageContent(Message $message): self
+    {
+        $message->html($this->mail->html ?? '')
+            ->text($this->mail->text ?? '');
+
+        foreach ($this->mail->attachments as $attachment) {
+            $message->attachData(
+                $attachment->file_data ?? $attachment->fileData ?? '',
+                $attachment->file_name ?? $attachment->filename ?? '',
+                ['mime' => $attachment->mime_type ?? $attachment->mime ?? '']
+            );
+        }
+
+        return $this;
+    }
+
+    private function setMessageRecipients(Message $message): self
+    {
```

Comment:
`$this->cc` should have priority over `$this->mail->cc`. Same for other recipients.